### PR TITLE
R4R: Suggested process changes

### PR DESCRIPTION
**Pull request by cwgoes**
Some changes to `README.md` for clarity, in particular upon how the spec process interacts with the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,27 +7,27 @@ This repository is for standards development of Interchain Standards. Initially
...
   * Describe the shape of the a potential solution
   * Identify challenges to this proposal
 - _**Entrance Criteria**_:
-  * Prose outlining the problem or need and the general shape of a solution in a PR to a `./spec/{{ .Spec.Number }}-{{ .Spec.Name }}/proposal.md` file in this repo. This file should contain:
-    1. Illustrative examples of usage
-    1. High-level API
-    1. Discussion of key algorithms, abstractions and semantics
-    1. Identification of potential “cross-cutting” concerns and implementation challenges/complexity
+  * Prose outlining the problem or need and the general shape of a solution in a PR to a `./{{ .Spec.Number }}-{{ .Spec.Name }}/proposal.md` file in this repo. This file should contain:
+    1. Expected set of specification users along with their goals and requirements
```
Comment (jackzampolin) [inline README.md]: What exactly does this mean? Slightly unclear IMO

File: README.md [hunk 2]
```diff
@@ -7,27 +7,27 @@ This repository is for standards development of Interchain Standards. Initially
 #### Stage 0 - `Strawman`
 - _**Purpose**_: Start the specification process
 - _**Entrance Criteria**_: [Open an issue](https://github.com/cosmos/ics/issues/new) on this repository with a short outline of your proposal
```
Comment (jackzampolin) [inline README.md]: Maybe also add here that issue must have a spec-name and spec-number? Or do we assign those later?

File: README.md (hunk 1 above)
Comment (cwgoes) [reply README.md]: A list of users, e.g. "Cosmos Zones" or "REST API Clients" (could include specific zones or wallets later on) - clarified the sentence.

File: README.md (hunk 2 above)
Comment (cwgoes) [reply README.md]: Let's require a name but not a number

**Comment by cwgoes** [comment]
I don't know how I feel about separate `proposal.md` and `spec.md` files - why not integrate them into one (which will evolve from a proposal into a spec, but still contain the design rationale)? For example, [EIP 1051](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1051.md).

Besides this process, I think we also want an ICS "spec spec" - a basic organizational template for all specs, a la https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md. I can work on this (in a separate PR).

File: README.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ This repository is for standards development of Interchain Standards. Initially
   * Precisely describe the syntax and semantics using formal spec language
 - _**Entrance Criteria**_:
   * Everything from stage 1
-  * Initial specification text in a PR to add a `./spec/{{ .Spec.Number }}-{{ .Spec.Name }}/spec.md` file
+  * Initial specification text in a PR to add a `./spec/{{ .Spec.Number }}-{{ .Spec.Name }}/spec.md` file.
```
Comment (cwgoes) [inline README.md]:
```suggestion
  * Initial specification text in a PR to add a `./spec/{{ .Spec.Number }}-{{ .Spec.Name }}/spec.md` file
```

**Comment by jackzampolin** [comment]
I'm cool with these changes @cwgoes, do we want anyone else to chime in here?

**Comment by cwgoes** [comment]
Merging for now, glad to address any additional commentary in a followup PR.

**Pull request merged**